.github/workflows/CD.yml

```diff
@@ -22,11 +22,26 @@ jobs:
         run: |
           python3 -m pip install --upgrade build
           python3 -m build
-      - name: Upload
+      - name: Upload PYPI
         if: steps.semantic-release.outputs.released == 'true'
         run: |
           python3 -m pip install twine
           python3 -m twine upload --repository pypi dist/* -u __token__ -p ${{ secrets.PYPI_TOKEN }}
+      - name: Setup NODE
+        uses: actions/setup-node@v3
+        with:
+          node-version: "20.x"
```

Comment:
Do you really need to impose this? What is the default node version on the runner?

Reply:
20.11.0 LTS